server/emails/templates/passwordReset.js

```diff
@@ -0,0 +1,31 @@
+import Mailgen from "mailgen";
+
+const mailGenerator = new Mailgen({
+  theme: "default",
+  product: {
+    name: "Authors Haven",
+    link: "#"
+  }
+});
+
+// Prepare email contents
+export const passwordResetTemplate = (firstName, lastName, resetUrl) => {
+  const email = {
+    body: {
+      name: `${firstName} ${lastName}`,
+      intro: `You have received this email because a password reset 
+        request for your account was received.`,
+      action: {
+        instructions: "Click the button below to reset your password:",
+        button: {
+          color: "#DC4D2F",
+          text: "Reset your password",
+          link: resetUrl
+        }
+      },
+      outro: `If you did not request a password reset
+        further action is required on your part.`
```

Comment:
This line is displaying this in the email sample you attached: `If you did not request a password resetfurther action is required on your part.`. Please add ` No` before the `further action`